tenable_io/client.py

```diff
@@ -80,42 +92,12 @@ def _ini_api(self):
 
     def _init_helpers(self):
         """
-        Initialize all helpers.
+        Initializes all helpers.
         """
         self.folder_helper = FolderHelper(self)
         self.policy_helper = PolicyHelper(self)
         self.scan_helper = ScanHelper(self)
 
-    def _retry(f):
-        """
-        Decorator to retry when TenableIORetryableException is caught.
-        :param f: Method to retry.
-        :return: A decorated method that implicitly retry the original method upon \
-        TenableIORetryableException is caught.
-        """
-        def wrapper(*args, **kwargs):
-            count = 0
-            retry = True
-            sleep_ms = 0
-
-            while retry:
-                retry = False
-                try:
-                    return f(*args, **kwargs)
-                except TenableIORetryableApiException as exception:
-                    count += 1
-
-                    if count <= TenableIOClient.MAX_RETRIES:
-                        retry = True
-                        sleep_ms += count * TenableIOClient.RETRY_SLEEP_MILLISECONDS
-                        logging.warn(u'RETRY(%d/%d)AFTER(%dms):%s' %
-                                     (count, TenableIOClient.MAX_RETRIES, sleep_ms, format_request(exception.response)))
-                        sleep(sleep_ms / 1000.0)
-                    else:
-                        raise TenableIOApiException(exception.response)
-
-        return wrapper
-
     def _error_handler(f):
         """
```

Comment:
Github is not allowing me to comment on the specific line, but `_error_handler` can now be simplified as there is no more need to classify error response into `TenableIORetryableApiException` (this assumes we can handle `501, 502, 503, 504` in the same manner as `429`. Moreover, `TenableIORetryableApiException` would also be obsolete.